languages/burmese.md

```diff
@@ -48,3 +48,11 @@ supported_apis:
 
 ---
 
+## Character encoding
+
+Burmese characters can be represented in Unicode or Zawgyi.
+Zawgyi is a variation on Unicode which interprets some code points differently but otherwise encodes code points in the same way with UTF-8 and other encodings.
```

Comment:
Maybe *as* not *with*?

> the same way as UTF-8.

Reply:
Looking at it more, I'll simplify further and just say "with UTF8 and others". I was trying to avoid saying that it used UTF-8 because I wasn't sure if UTF-8 implies Unicode code points. Wikipedia doesn't seem to tie them together too carefully though so I think it's better to simplify